applications/tari_console_wallet/src/ui/app.rs

```diff
@@ -153,6 +153,7 @@ impl<B: Backend> App<B> {
 
     pub fn on_tick(&mut self) {
         Handle::current().block_on(self.app_state.update_cache());
+        Handle::current().block_on(self.app_state.check_connectivity());
```

Comment:
This is not ideal. Connectivity state changes are all accompanied by events and so can be handled in the monitor when they arrive. The event based architecture means we do not need a complex polling + cooldown approach.

Reply:
Have removed the polling + cooldown.